website/docs/r/lex_bot.html.markdown

```diff
@@ -0,0 +1,159 @@
+---
+layout: "aws"
+page_title: "AWS: aws_lex_bot"
+sidebar_current: "docs-aws-resource-lex-bot"
+description: |-
+  Provides an Amazon Lex bot resource.
+---
+
+# Resource: aws_lex_bot
+
+Provides an Amazon Lex Bot resource. For more information see
+[Amazon Lex: How It Works](https://docs.aws.amazon.com/lex/latest/dg/how-it-works.html)
+
+## Example Usage
+
+```hcl
+resource "aws_lex_bot" "order_flowers_bot" {
+  abort_statement {
+    message {
+      content      = "Sorry, I am not able to assist at this time"
+      content_type = "PlainText"
+    }
+  }
+
+  child_directed = false
+
+  clarification_prompt {
+    max_attempts = 2
+
+    message {
+      content      = "I didn't understand you, what would you like to do?"
+      content_type = "PlainText"
+    }
+  }
+
+  create_version              = false
+  description                 = "Bot to order flowers on the behalf of a user"
+  idle_session_ttl_in_seconds = 600
+
+  intent {
+    intent_name    = "OrderFlowers"
+    intent_version = "1"
+  }
+
+  locale           = "en-US"
+  name             = "OrderFlowers"
+  process_behavior = "BUILD"
+  voice_id         = "Salli"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `abort_statement` - (Required) The message that Amazon Lex uses to abort a conversation. Attributes 
+are documented under [statement](#statement).
+* `child_directed` - (Required) By specifying true, you confirm that your use of Amazon Lex is related 
+to a website, program, or other application that is directed or targeted, in whole or in part, to 
+children under age 13 and subject to COPPA.
+
+By specifying false, you confirm that your use of Amazon Lex is not related to a website, program, or 
+other application that is directed or targeted, in whole or in part, to children under age 13 and 
+subject to COPPA.
+
+If your use of Amazon Lex relates to a website, program, or other application that is directed in 
+whole or in part, to children under age 13, you must obtain any required verifiable parental consent 
+under COPPA. For information regarding the use of Amazon Lex in connection with websites, programs, 
+or other applications that are directed or targeted, in whole or in part, to children under age 13, 
+see the [Amazon Lex FAQ](https://aws.amazon.com/lex/faqs#data-security).
```

Comment:
We should probably remove a lot of this text, but add a link to the corresponding text in the AWS documentation.

Reply:
I'll keep the first paragraph and add a link to the putbot api docs plus the faq link. I originally put the complete statement here due to the sensitive / legal nature of this setting.